doc: add notes about CSRF and Form requirements (#98)
zenstruck/messenger-monitor-bundle: README.md

> Install `lorisleiva/cron-translator` (`composer require lorisleiva/cron-translator`) to display
> friendlier CRON values for your scheduled tasks.

> > [!NOTE]
> > Install `symfony/form` (`composer require symfony/form`) for the CSRF support when doing Ajax calls.
> > Necessary if when using symfony/scheduler is used.

Reviewer: ```suggestion
> [!NOTE]
>`symfony/form` (`composer require symfony/form`) is required for the UI.
```

And let's move this to the top of this section - right after `## User Interface`
